docs/manual/getting_started.rst

```diff
@@ -246,19 +333,19 @@ using the Hello World example is explained in this section.
 Environnement variable updates
 ''''''''''''''''''''''''''''''
 
-On Windows, to run Eclipse Cyclone DDS executables, the required libraries
-(like ``ddsc.dll``) need to be available to the executables. Normally,
-these libraries are installed in system default locations and it works
-out-of-the-box. However, if they are not installed in those locations,
-the library search path must be changed. This can be achieved by
-executing the command:
+To run Eclipse Cyclone DDS executables on Windows, the required libraries
+(like ``ddsc.dll``) must be available to the executables. Typically,
+these libraries are installed in system default locations and work
```

Comment:
We should specify default locations. Otherwise users will have to `grep` for them in the source or  via their OS searching after installing.